index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="main-container">
+            <div class="header-container">
+                <div class="my-logo">
+                  <h1 class="logo">My Logo</h1>
+                </div>
+                <div class="hambuger">
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="img-BG">
+        <div class="main-container">
+            <div class="portfolio-description">
+                <div class="name">
+                    <h1>I'm Ifeanyi, Glad to see you!</h1>
+                </div>
+                <div class="my-qualities">
+                  <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="lets-connect">
+                  <p>LET’S CONNECT</p>
+                </div>
+                <div class="social-media">
+                    <div><img src="images/twitter-icon.svg" alt=""></div>
+                    <div><img src="images/Linkedin icon.svg" alt=""></div>
+                    <div><img src="images/M-logo.svg" alt=""></div>
+                    <div><img src="images/github-icon.svg" alt=""></div>
+                    <div><img src="images/index-finger.svg" alt=""></div>
+                </div>
+            </div>
```

Comment:
- [ ] [REQUIRED] You should use a list (ul) and the nav tag here. Since it is a list of logos and they take visitors to your social pages. Also kindly put each icon in an anchor tag.
- [ ]  [REQUIRED] Kindly increase the space between the icons as present in the template